lib/python/picongpu/plugins/plot_mpl/energy_histogram_visualizer.py

```diff
@@ -37,15 +37,18 @@ def _create_plt_obj(self, ax):
         Implementation of base class function.
         Turns 'self.plt_obj' into a matplotlib.pyplot.plot object.
         """
-        bins, counts = self.data
+        counts, bins = self.data
```

Comment:
great fix - just a suggestion, `bins` is actually bin borders

Reply:
yep, that should be documented in the data API, not too important in the matplotlib backend to mention again since it's not visible to users.